shell/driver.postgresql/db-load

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# db-load -- Loads file to the PostgreSQL database configured for a DeepDive application
```

Comment:
I think you need to add this new `load` operation to `load-db-driver.sh` as well.